elide-core/src/main/java/com/yahoo/elide/core/EntityDictionary.java

```diff
@@ -1415,9 +1490,11 @@ private boolean isValidParameterizedMap(Map<?, ?> values, Class<?> keyType, Clas
      * @param entityClass the class to bind.
      */
     private void bindIfUnbound(Class<?> entityClass) {
-        if (! entityBindings.containsKey(lookupEntityClass(entityClass))) {
-            bindEntity(entityClass);
+        if (lookupBoundClass(entityClass) != null) {
+            return;
         }
+
+        bindEntity(entityClass);
```

Comment:
```Java
if (lookupBoundClass(entityClass) == null) {
  bindEntity(entityClass);
}
```
Might be simpler or more straight forward?

Reply:
:+1: